docs/Using-Operations-Service.md

```diff
@@ -559,3 +559,24 @@ When the app is launched via a deeplink, preserve the data from the deeplink and
 
 - Authorizing the ProximityCheck
   When authorizing, the SDK will by default add `timestampSigned` to the `ProximityCheck` object. This timestamp indicates when the operation was signed.
+
+### PACUtils
+- For convenience, utility class for parsing and extracting data from QR codes and deeplinks used in the PAC (Proximity Anti-fraud Check), is provided.
+```kotlin
+    /** Data payload which is returned from the parser */
+    data class PACData(
+
+        /** The ID of the operation associated with the TOTP */
+        @SerializedName("oid")
```

Comment:
I would remote the `@SerialzedName` from the docs, it's confusing and not necessary